hs_core/tests/api/native/test_get_resource_file.py

```diff
@@ -14,31 +13,28 @@ def setUp(self):
         super(TestGetResourceFile, self).setUp()
         self.group, _ = Group.objects.get_or_create(name='Hydroshare Author')
         self.user = hydroshare.create_account(
-            'jamy2@gmail.com',
-            username='jamy2',
-            first_name='Tian',
-            last_name='Gan',
+            'testuser@gmail.com',
+            username='testuser',
+            first_name='test',
+            last_name='user',
             superuser=False,
             groups=[]
         )
 
         self.res = hydroshare.create_resource(
-            resource_type='GenericResource',
+            resource_type='CompositeResource',
             owner=self.user,
-            title='My resource',
-            metadata=[],
+            title='Test File'
         )
 
-        open('myfile.txt', "w").close()
-        self.file = open('myfile.txt', 'rb')
-
+        test_file = open('test1.txt', 'w')
+        test_file.write("Test text file in test1.txt")
+        test_file.close()
+        self.file = open('test1.txt', 'rb')
         hydroshare.add_resource_files(self.res.short_id, self.file)
 
     def tearDown(self):
         super(TestGetResourceFile, self).tearDown()
-        User.objects.all().delete()
-        GenericResource.objects.all().delete()
-        ResourceFile.objects.all().delete()
```

Comment:
Are these getting deleted elsewhere?

Reply:
@sblack-usu I believe these delete() statements are not needed since TestCase is run in a db transaction so all db related cleanup is not needed, hence I removed these delete() statements.